realm/realm-library/src/objectServer/java/io/realm/internal/network/LookupUserIdResponse.java

```diff
@@ -127,21 +133,21 @@ public boolean isAdmin() {
 
     public Map<String, String> getMetadata() { return metadata; }
 
-    private static Map<String, String> jsonToMap(JSONObject json) throws JSONException {
-        Map<String, String> map = new HashMap<>();
-        if(json != JSONObject.NULL) {
-            map = toMap(json);
-        }
-        return map;
-    }
+    public Map<String, String> getAccounts() { return accounts; }
```

Comment:
any tests that covers `getAccounts ` (and by extension `jsonToMap`)

Reply:
Yes: https://github.com/realm/realm-java/pull/5313/files#diff-5ad3dd1db70a9dde54cfd2af71abbbe7R625